api_tests/e2e/rfc003/btc_eth-erc20/test.js

```diff
@@ -154,12 +148,6 @@ describe("RFC003: Bitcoin for ERC20", () => {
             alpha_ledger_redeem_identity: null,
         };
 
-        logger.info(
```

Comment:
I am not sure how wise it is to remove all logging

Reply:
I am happy to add them back in, but I believe they're not being used all that much. @comit-network/rust-devs: thoughts?

Reply:
I think we no longer need it. Maybe we can add some logging once the tests are consolidated with #767. That would allow to do it without a lots of repetition.

Reply:
ok, please update #767